easybuild/easyconfigs/t/TensorFlow/TensorFlow-2.0.0-fosscuda-2019b-Python-3.7.4.eb

```diff
@@ -0,0 +1,107 @@
+easyblock = 'PythonBundle'
+
+name = 'TensorFlow'
+version = '2.0.0'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://www.tensorflow.org/'
+description = "An open-source software library for Machine Intelligence"
+
+toolchain = {'name': 'fosscuda', 'version': '2019b'}
+toolchainopts = {'usempi': True}
+
+builddependencies = [
+    ('Bazel', '0.26.1'),
+    ('protobuf', '3.7.1'),
+    # git 2.x required, see also https://github.com/tensorflow/tensorflow/issues/29053
+    ('git', '2.23.0'),
+]
+dependencies = [
+    ('Python', '3.7.4'),
+    ('SciPy-bundle', '2019.10', '-Python-3.7.4'),
+    ('h5py', '2.9.0'),
```

Comment:
@Flamefire You'll need to add `versionsuffix` for `h5py` (and also use it for `SciPy-bundle` rather than hardcoding it:

```python
dependencies = [
    ('Python', '3.7.4'),
    ('SciPy-bundle', '2019.10', versionsuffix),
    ('h5py', '2.9.0', versionsuffix),

Reply:
Already done locally. Waiting for #9248 to push the changes

Reply:
:+1: